src/components/login/SignIn.tsx

```diff
@@ -22,8 +29,26 @@ export const SignIn = ({ changeMode }: SignInProps) => {
   const [password, setPassword] = useState('');
   const [error, setError] = useState('');
   const [isLoading, setLoading] = useState(false);
+  const [visible, setVisible] = useState('password');
   const { authenticate } = useAuth();
 
+  const toast = useToast();
+
+  const toastErrorTemplate = () => {
+    const id = 'uniqueToast';
+    if (!toast.isActive(id)) {
+      toast({
+        id,
+        title: 'Error',
+        description: error,
+        duration: 3000,
+        isClosable: true,
+        status: 'error',
+        position: 'top-right',
```

Comment:
we can also try to create some utils/hook to use unique toasts, you can follow on that way but after merge i'll create another abstraction to make it simpler.

Reply:
can you give an example?